qiskit_nature/circuit/library/ansatzes/utils/fermionic_excitation_generator.py

```diff
@@ -128,6 +124,56 @@ def generate_fermionic_excitations(
         The list of excitations encoded as tuples of tuples. Each tuple in the list is a pair. The
         first tuple contains the occupied spin orbital indices whereas the second one contains the
         indices of the unoccupied spin orbitals.
+
+    Examples:
+        Generate excitations with basic inputs.
+
+        .. jupyter-execute::
+
+            from qiskit_nature.circuit.library.ansatzes.utils.fermionic_excitation_generator import (
+                generate_fermionic_excitations
+            )
+
+            num_excitations = 1
+            num_spin_orbitals = 4
+            num_particles = [1,1]
+
+            excitations = generate_fermionic_excitations(
+                num_excitations, num_spin_orbitals, num_particles
+            )
```

Comment:
```suggestion
            excitations = generate_fermionic_excitations(1, 4, (1, 1))
```
I think for these examples we can inline the arguments.
Also note that `num_particles` should be a tuple and not a list.

Same holds below but I will not comment on each occurrence now.

Reply:
I actually think it is a clearer introductory example to write them all out explicitly. Maybe I can leave the first basic example written out in full and then inline them for the other examples?

Reply:
In that case I would inline it:
```
excitations = generate_fermionic_excitations(
    num_excitations=1,
    num_spin_orbitals=4,
    num_particles=(1, 1),
)
```